templates/features.html

```diff
@@ -191,7 +191,7 @@ <h3>{% trans %}Plotting{% endtrans %}</h3>
                 </figure>
                 <figure>
                     <img src="/static/images/main_features_examples/plotting2.png"
-                         alt="{% trans %}SymPy plot of x, x^2, and x^3 from x = -5 to 5{% endtrans %}" />
+                         alt="{% trans %}SymPy plot of x, x^2, and x^3, x from -5 to 5{% endtrans %}" />
```

Comment:
This was correct before.

Reply:
Yes, but this is the way it's written for the other examples. That's why I changed it.

Reply:
But this is not the important part of the PR. I completely translated the website to Spanish.

Reply:
Thanks @nicoguaro for this PR! This looks like a lot of work.

I think @asmeurer is right though. This was correct before. Maybe the other examples should be changed to match this.

Reply:
I changed this because the other we more in quantity. I can go and change the others, but, so we are not wasting time would you go to the website and double-check before?

Reply:
I took a look at the website and I think that they can all be changed. It should be  "from x = -5 to 5" as it was in this case.

Reply:
What about this one?

    SymPy 3D parametric surface plot of x=cos(u+v), y=sin(u-v), z=u-v, u and v from -5 to 5

Reply:
> What about this one?

I don't know if that's the current text or your suggested change but either way it seems fine.

Reply:
It is done now.

I fixed my "mistake" and made the captions uniform. And also added the translation for those new changes.